apache-rat-core/src/main/java/org/apache/rat/config/exclusion/StandardCollection.java

```diff
@@ -0,0 +1,306 @@
+    /**
+     * The .DS_Store files MAC computer.
+     */
+    MAC("The .DS_Store files MAC computer.",
+            Collections.singletonList("**/.DS_Store"), null, null),
+    /**
+     * The files and directories created by Maven build system based project.
+     */
+    MAVEN("The files and directories created by Maven build system based project.",
+            Arrays.asList(//
+                    "target/**", //
+                    "cobertura.ser", //
+                    "**/MANIFEST.MF", // a MANIFEST.MF file cannot contain comment lines. In other words: It is not possible, to include a license.
+                    "release.properties", //
+                    ".repository", // Used by Jenkins when a Maven job uses a private repository that is "Local to the workspace"
+                    "build.log", // RAT-160: until now maven-invoker-plugin runs create a build.log that is not part of a release
+                    ".mvn/**", // Project configuration since Maven 3.3.1 which contains maven.config, jvm.config, extensions.xml
+                    "pom.xml.releaseBackup"), null, null),
+    /**
+     * The files and directories created by a Mercurial source code control based tool.
+     */
+    MERCURIAL("The files and directories created by a Mercurial source code control based tool.",
+            Arrays.asList("**/.hg/**", ".hgignore"), null, new HgIgnoreProcessor()),
+    /**
+     * The set of miscellaneous files generally left by editors and the like.
+     */
+    MISC("The set of miscellaneous files generally left by editors and the like.",
+            Arrays.asList("**/*~", "**/#*#", "**/.#*", "**/%*%", "**/._*"),
+            null, null),
+    /**
+     * The files and directories created by an MKS source code control based tool.
+     */
+    MKS("The files and directories created by an MKS source code control based tool.",
+            Collections.singletonList("**/project.pj"), null, null),
+    /**
+     * The files and directories created by a RCS source code control based tool.
+     */
+    RCS("The files and directories created by a RCS source code control based tool.",
+            Collections.singletonList("**/RCS/**"), null, null),
+    /**
+     * The files and directories created by a SCCS source code control based tool.
+     */
+    SCCS("The files and directories created by a SCCS source code control based tool.",
+            Collections.singletonList("**/SCCS/**"), null, null),
+    /**
+     * The files and directories created by a Serena Dimensions V10 change control system based tool.
+     */
+    SERENA_DIMENSIONS_10("The files and directories created by a Serena Dimensions V10 change control system based tool.",
+            Collections.singletonList("**/.metadata/**"), null, null),
+    /**
+     * A standard collection of generally accepted patterns to ignore.
+     */
+    STANDARD_PATTERNS("A standard collection of generally accepted patterns to ignore.", null, null, null),
```

Comment:
Should we document that both standard-things are empty?! or add a comment why they are necessary? By reading over it I asked myself what they are good for.